dd-java-agent/instrumentation/netty-4.0/src/main/java/datadog/trace/instrumentation/netty40/server/HttpServerResponseTracingHandler.java

```diff
@@ -28,12 +30,12 @@ public void write(final ChannelHandlerContext ctx, final Object msg, final Chann
     } catch (final Throwable throwable) {
       Tags.ERROR.set(span, Boolean.TRUE);
       span.log(Collections.singletonMap(ERROR_OBJECT, throwable));
-      Tags.HTTP_STATUS.set(span, 500);
+      NettyUtils.setServerSpanHttpStatus(span, 500);
```

Comment:
This is still assuming a 500 when we don't know for sure that the server is actually returning that to the client -- Or do we?

Reply:
Updated